_config.yml

```diff
@@ -11,6 +11,19 @@ title: "Lesson Title"
 # Contact email address.
 email: lessons@software-carpentry.org
 
+#Metadata
+# Here you can add metadata to describe your lesson so that people and search engines can understand what it's about. Please try to use fields from the Schema.org CreativeWork type - https://schema.org/CreativeWork
+description:    "This is the template description. Keep me brief (2-3 sentences)"
+keywords:       GitHub, Forking, Collaborative                  # see: https://schema.org/keywords
+audience:       [PostDoc students, Early Career Researchers]    # see: https://schema.org/audience
+license:        "https://creativecommons.org/licenses/by/3.0/"  # see: schema.org/license
+author:         [Homer Simpson, Ned Flanders]                   # see: https://schema.org/author
+contributor:    [Barney Gumball, Dr Nick Riviera]               # see: https://schema.org/contributor
```

Comment:
Is there an `editor` role? We currently identify the lesson maintainers as editors for publication purposes when getting DOIs from Zenodo, and list everyone who's had a commit merged as an author.